src/openloco/openloco.vcxproj

```diff
@@ -94,6 +94,7 @@
     <ClCompile Include="windows\CompanyWindow.cpp" />
     <ClCompile Include="windows\constructionwnd.cpp" />
     <ClCompile Include="windows\EditKeyboardShortcut.cpp" />
+    <ClCompile Include="windows\Error.cpp" />
```

Comment:
This filename was changed to title case, but the filename in the repo is actually in lowercase still.

Reply:
I just renamed it in VS, should I have done something different?

Reply:
Windows file systems are case-insensitive, so the change may just not have been picked up by git. In any case, due to this same case-insensitivity, there's no issue building on Windows, either. We'll get around to renaming it, eventually.